go/runtime/bundle/component_test.go

```diff
@@ -0,0 +1,42 @@
+package bundle
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/oasisprotocol/oasis-core/go/runtime/bundle/component"
+)
+
+func TestComponentValidation(t *testing.T) {
+	require := require.New(t)
+
+	comp := Component{}
```

Comment:
```suggestion
	var comp Component
```
Zero values should be defined like this, its more readable.

Could also use table-test. I would add comment at the top that this section only tests names.

Reply:
Fixed.